Core/include/Acts/MagneticField/ConstantBField.hpp

```diff
@@ -21,7 +21,7 @@ class ConstantBField final {
  public:
   struct Cache {
     /// @brief constructor with context
-    Cache(std::reference_wrapper<const MagneticFieldContext> /*mcfg*/) {}
```

Comment:
I like, we used this reference wrapping only for the context mechanism, happy when this goes.

Reply:
Yeah we still need it when we want to **store** a reference to the context object, which we sometimes do in our state objects, but that's not problematic.